agent/util-scripts/pbench-tool-meister-start

```diff
@@ -432,15 +431,14 @@ class BaseServer:
         if len(pairs) == 2:
             # Separate bind/connecting ip:port
             self.host, self.port = pairs[1]
+            self._repr = f"{self.name} - {self.bind_host}:{self.bind_port} / {self.host}:{self.port}"
         else:
             assert len(pairs) == 1, "Logic bomb!  unexpected pairs, {pairs!r}"
             self.host, self.port = pairs[0]
+            self._repr = f"{self.name} - {self.host}:{self.port}"
 
     def __repr__(self):
-        return f"{self.host}:{self.port} / {self.bind_host}:{self.bind_port}"
-
-    def __str__(self):
-        return f"{self.host}:{self.port}"
+        return self._repr
```

Comment:
:-D

I was thinking that the code would decide what to print dynamically, if/when it was called upon to actually print something, rather than use an attribute....  c'est la vie.

Reply:
It seems silly to dynamically create the string when it is really fixed a creation time.

Reply:
It's mostly a space vs. speed trade-off -- why create and store the value if it's never actually needed?  If it's only needed on an error or diagnostic path, why pay the extra cost in the normal case?